utils/canutils/Makefile

```diff
@@ -8,11 +8,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=canutils
-PKG_RELEASE=2
+PKG_RELEASE=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/linux-can/can-utils
-PKG_SOURCE_VERSION:=0e3ff3b3157e456d4b6343f5d4b42ef692bce538
+PKG_SOURCE_VERSION:=cb33a55720716cbe01e6025a2bda74a1b7e492d3
```

Comment:
+1 for this. 
The combination of PKG_VERSION-PKG_REVISION should be always growing. Using git hash as the main version does not guarantee that.
Using e.g. source date as the prefix for the mai version will work, and is the "standard practice" in LEDE git based core packages.

You can use the automatics in LEDE and Openwrt masters, and just define PKG_SOURCE_DATE. When that is defined, it will be included in an automatically created PKG_VERSION. 

Good example is uhttpd:

https://git.lede-project.org/?p=source.git;a=blob;f=package/network/services/uhttpd/Makefile;hb=HEAD
```
PKG_RELEASE:=1
PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL=$(LEDE_GIT)/project/uhttpd.git
PKG_SOURCE_DATE:=2016-10-25
PKG_SOURCE_VERSION:=1628fa4b34aa143187353f81e8001b9a15286bda
PKG_MIRROR_HASH:=2ac4ba8dc0b349d72174aac9ff693a73a214295a9890fe3d2a8eedcad54d06e3
```
leads to ` uhttpd_2016-10-25-1628fa4b-1 `